tools/remove_fixture

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Not sure I would add tooling in the test folder. It would be inconvenient to have a long path, and to be fair I would expect only code to be below `assisted-events-scrape` subfolder.
However we can think a way of better organizing them, instead of having a "laundry drawer" of tools. Possibly `tools/testing/remove_fixture.sh`, WDYT?

Reply:
Yeah, I only refer to scripts that are tightly-coupled with the environment. Anything that's generic/agnostic can stay in its place

Reply:
Alright, I'll give it a chance - but it still seems awkward to me (also having a long path for tools it kind of defeats the purpose).
We'll see how it goes